deltachat-ios/Controller/WebxdcViewController.swift

```diff
@@ -116,6 +116,8 @@ class WebxdcViewController: WebViewViewController {
         config.mediaTypesRequiringUserActionForPlayback = []
         config.allowsInlineMediaPlayback = true
 
+        config.websiteDataStore = WKWebsiteDataStore.default()
```

Comment:
i think, this is not even needed, [websiteDataStore documentation](https://developer.apple.com/documentation/webkit/wkwebviewconfiguration/1395661-websitedatastore) says:

> If you don’t assign a value to this property, the configuration object uses the default data store object to store data persistently

i also tested with the test_storage.xdc, it works with and without this pr.

so not sure if we should make that explicit.

Reply:
Hm, ok, then let's close this PR and come back to it, if issues appear